### ISSUE TITLE
More details on the pop up

### ISSUE THREAD

**Issue by elolelo** [IneffableKoD/cov19testmap#23]
It could be useful to know prior to visiting any testing center what requirements are (fee,doctor's referral note? etc)  also does one need to book an appointment first or is it a drive thru? 

Such general info could be helpful, maybe it's worth adding with the info that shows on the pop up of each point of testing facilitity - don't you think?

**Comment by IneffableKoD** [comment]
Yes, that is a great idea! Do you want to propose how that could look?

The reason I kept it relatively simple so far it that I didn't find much information and just packed what I had into comments.

**Comment by elolelo** [comment]
I have submitted a pull request that shows my proposal. I struggled with finding an Icon for ZAF  hospitals - please be wary of this.
